tests/data/estimation/calibration_factor_combined.csv

```diff
@@ -0,0 +1,9 @@
+period,strata,calibration_group,sampled,auxiliary,design_weight,calibration_factor
+202401,101,A,0,300,2.0,1.2
+202401,101,A,1,100,2.0,1.2
+202401,102,A,1,100,1.5,1.2
```

Comment:
Yes this was a mistake in this dataset. Design weight is only here because I thought we needed to recalculate the a weight by group for the g weight so I wanted to check it wasn't overwritten, rather than testing the how a weights are calculated which is done in a separate test